src/widget/sidebar.c

```diff
@@ -76,7 +79,6 @@ static void button_empire(int param1, int param2);
 static void button_mission_briefing(int param1, int param2);
 static void button_rotate_north(int param1, int param2);
 static void button_rotate(int clockwise, int param2);
-static void button_game_speed(int is_down, int param2);
```

Comment:
note - `button_game_speed(int is_down, int param2)` is now in sidebar_filler.c - i know it looks odd to split it out from the other `button_` sidebar functions, but to me doing this makes sense when organization is looked at from the perspective of "original game" vs "julius additions".

thoughts?

Reply:
It doesn't look odd at all. Buttons, click handlers, etc, should go in the code that uses it. In the end, `sidebar.c` shouldn't know anything about buttons for game speed, and `sidebar_filler.c` should be an entirely self-sufficient piece of code. Just like the minimap, which handles its own clicks.